doc/sphinx/proof-engine/tactics.rst

```diff
@@ -606,6 +611,10 @@ Applying theorems
       when the instantiation of a variable cannot be found
       (cf. :tacn:`eapply` and :tacn:`apply`).
 
+.. flag:: Debug Tactic Unification
+
+   Enables printing traces of unification steps in tactic unification.
+   Tactic unification is used in tactics such as :tacn:`apply` and :tacn:`rewrite`.
```

Comment:
(also known as "the old unification engine")

Reply:
> (also known as "the old unification engine")

You're referring to "tactic unification"?  Not sure what change you're suggesting or it's just a side comment.

Reply:
Yes, but this is just a side comment :wink: